pkgs/by-name/ca/cano/package.nix

```
@@ -4,14 +4,14 @@
 , gnumake
 , ncurses
 }:
-stdenv.mkDerivation (finalAttrs: {
+stdenv.mkDerivation (finalAttrs: rec {
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
```

Use `"v${finalAttrs.version}"`